tests/expressions_case/tests.py

```diff
@@ -1301,3 +1301,9 @@ def test_invalid_when_constructor_args(self):
             When(condition=object())
         with self.assertRaisesMessage(TypeError, msg):
             When()
+
+    def test_empty_q_object(self):
+        """A Q object with an empty condition should be rejected as the
+        conditional expression in a Case()."""
```

Comment:
I think we prefer a different indentation style in docstrings, i.e.:
```python
"""
A Q object with an empty condition should be rejected as the
conditional expression in a Case().
"""
```

Reply:
Good point, thanks for the correction.